modules/data.land/inst/doanload_NEON_multi_site.R

```diff
@@ -0,0 +1,111 @@
+#' Title generalized for download NEON files of multiSites. and automatically detect the most recent available NEON data and if data is problematic, try next year.
```

Comment:
Fix spelling error in filename

Reply:
fixed

Reply:
filename still listed as `doadload_NEON_multi_site.R` please fix via `git mv`